internal/installation/receipt/receipt.go

```diff
@@ -42,6 +43,16 @@ func Load(path string) (index.Receipt, error) {
 	return indexscanner.ReadReceiptFromFile(path)
 }
 
+// CanonicalName returns the index and plugin name from a receipt.
+// The index name is omitted if it is the default index.
+func CanonicalName(receipt index.Receipt) string {
```

Comment:
this is actually not a `CanonicalName`.
Canonical name of a `foo` plugin is `default/foo`.

If anything, this is a `DisplayName` method.

Therefore, I recommend we define this in `cmd/` package.
Also keep in mind this will come up in many other use cases e.g.:

- `krew search`, similarly listing plugins
- `krew install`/`upgrade` calling `krew update` implicitly which lists new/updated plugins in the index(es).

Reply:
discussed offline, we've decided to move this to cmd/.

- displaying is a CLI concern and all the functionality in there would use this
- we decided not to add it on Receipt type, to keep the pkg/index (one of our two public pkgs) small and type-only